custom-connectors/My Acclaro/apiDefinition.swagger.json

```diff
@@ -0,0 +1,13329 @@
+{
+  "swagger":"2.0",
+  "info":{
+    "version":"1.0.0",
+    "title":"My Acclaro",
+    "description":"My Acclaro allows you to seamlessly connect to the Acclaro client portal to submit your translation projects directly.",
+    "contact":{
+      "name": "My Acclaro Support",
+      "url": "https://www.acclaro.com/support/",
+      "email": "support@acclaro.com"
+    }
+  },
+  "host":"my.acclaro.com",
+  "basePath":"/api/v2",
+  "schemes":[
+    "https"
+  ],
+  "consumes":[
+    
+  ],
+  "produces":[
+    "application/json"
+  ],
+  "paths":{
+    "/info/version":{
+      "get":{
+        "summary":"Get system information",
+        "tags":[
+          "info"
+        ],
+        "description":"Returns a variety of helpful system information including version of API source code in use, service endpoint and last date of database synchronization.",
+        "operationId":"GetSystemInformation",
+        "parameters":[
+          
+        ],
+        "responses":{
+          "200":{
+            "description":"200",
+            "schema":{
+              "type":"object",
+              "properties":{
+                "success":{
+                  "type":"boolean",
+                  "description":"success"
+                },
+                "data":{
+                  "type":"object",
+                  "properties":{
+                    "version":{
+                      "type":"string",
+                      "description":"version"
+                    },
+                    "endpoint":{
+                      "type":"string",
+                      "description":"endpoint"
+                    },
+                    "sitedescription":{
+                      "type":"string",
+                      "description":"sitedescription"
```

Comment:
@msanz-acclaro All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
this has been addressed